.github/workflows/nightly-ci.yml

```
@@ -412,6 +413,29 @@ jobs:
       image_tag_suffix: '-nightly'
     secrets: inherit
 
+  sglang-h100-test:
+    name: sglang-runtime # This name overlaps with other sglang jobs to group them in the UI
+    needs: [sglang-build, resolve-source-sha, compute-release-mode]
+    if: ${{ needs.compute-release-mode.outputs.run_tests == 'true' }}
+    uses: ./.github/workflows/shared-test.yml
+    with:
+      dd_env: nightly
+      dd_flaky_retry_enabled: 'false'
```

actionlint (1.7.12)

[error] input "dd_flaky_retry_enabled" is typed as string by reusable workflow "./.github/workflows/shared-test.yml". bool value cannot be assigned

(expression)